src/xngin/apiserver/routers/experiments/test_experiments_api.py

```diff
@@ -222,6 +222,61 @@ async def test_get_assignment_for_participant_with_apikey_online(xngin_session,
     assert assignment.experiment.stopped_assignments_reason is None
 
 
+async def test_get_assignment_for_participant_with_apikey_mab_online(xngin_session, testing_datasource, client_v1):
+    """Test endpoint that gets an assignment for a participant via API key."""
+    online_experiment = await insert_experiment_and_arms(
+        xngin_session,
+        testing_datasource.ds,
+        experiment_type=ExperimentsType.MAB_ONLINE,
+    )
+
+    response = client_v1.get(
+        f"/experiments/{online_experiment.id!s}/assignments/1",
+        headers={constants.HEADER_API_KEY: testing_datasource.key},
+    )
+    assert response.status_code == 200
+    parsed = GetParticipantAssignmentResponse.model_validate_json(response.text)
+    assert parsed.experiment_id == online_experiment.id
+    assert parsed.participant_id == "1"
+    arms_map = {arm.id: arm.name for arm in online_experiment.arms}
+    assert parsed.assignment is not None
+    assert parsed.assignment.arm_name == arms_map[str(parsed.assignment.arm_id)]
+    assert not parsed.assignment.strata
+    assert parsed.assignment.observed_at is None
+    assert parsed.assignment.outcome is None
+    assert parsed.assignment.context_values is None
+    assert parsed.assignment.current_mu is None
+    assert parsed.assignment.current_covariance is None
+    assert parsed.assignment.current_alpha is None
+    assert parsed.assignment.current_beta is None
+
+    # Test that we get the same assignment for the same participant.
+    response2 = client_v1.get(
+        f"/experiments/{online_experiment.id!s}/assignments/1",
+        headers={constants.HEADER_API_KEY: testing_datasource.key},
+    )
+    assert response2.status_code == 200
+    assert response2.json() == response.json()
+
+    # Make sure there's only one db entry.
+    assignment = (
+        await xngin_session.scalars(select(tables.Draw).where(tables.Draw.experiment_id == online_experiment.id))
+    ).one()
+    assert assignment.participant_id == "1"
+    assert assignment.arm_id == str(parsed.assignment.arm_id)
+
+    # Verify no update to experiment lifecycle info.
+    assert assignment.observed_at is None
+    assert assignment.outcome is None
+    assert assignment.context_vals is None
+    assert assignment.current_mu is None
+    assert assignment.current_covariance is None
+    assert assignment.current_alpha is None
+    assert assignment.current_beta is None
```

Comment:
Should this test and the creation asserts higher up be passing if everything is None? 
The fake data created via `make_createexperimentrequest_json()` uses NORMAL as prior, so I'd expect mu & covariance. Ah, I commented about that missing in `get_existing_assignment_for_participant`. 

Looks like trying to pass back all the params with each assignment is causing some headaches; can we avoid that addition instead?